worker/categorize.py

```diff
@@ -387,8 +376,7 @@ def _get_best_keywords(self, communities, original_keywords):
             best_original_keywords = keyword_counts.most_common(2)
             best_keywords.update([keyword[0] for keyword in best_original_keywords])
 
-        return [keyword.title() for keyword in best_keywords] if best_keywords else [""]
-
+        return [keyword.title() for keyword in best_keywords]
```

Comment:
It just won't go into a for loop when appending categories. Since I check for .categorized now instead, we don't need to create a NULL category anymore.